src/App.js

```diff
@@ -39,14 +39,17 @@ const personas = [
   },
 ]
 
+// margin-left: ${({ showSelector }) => (showSelector ? '400px' : '0px')};
 const FullScreenGrid = styled(Grid)`
   height: 100lvh;
```

Comment:
this currently still introduces a vertical scroll bar because of the border being beyond the width. Two possible solutions:

1. change `box-sizing` to `border-box` and then set `width` to `calc(100lvw - 430px)`. This would be my recomendation unless there's something I've missed. I have tested this in the browser
2. change height to also be dynamic and adjust to when the border is added

Reply:
most of it should be covered, I hope:) thanks for a comment.